cli.js

```diff
@@ -23,44 +30,36 @@ Options:
 
 const options = docopt(doc, { version: '0.0.1' });
```

Comment:
y not get the version from package.json?

Reply:
done